crates/sncast/src/starknet_commands/verify/mod.rs

```diff
@@ -85,10 +94,26 @@ pub async fn verify(
         .parent()
         .ok_or(anyhow!("Failed to obtain workspace dir"))?;
 
+    let contract_identifier = match (verify.class_hash, verify.contract_address) {
+        (Some(class_hash), None) => ContractIdentifier::ClassHash {
+            class_hash: class_hash.to_fixed_hex_string(),
+        },
+        (None, Some(contract_address)) => ContractIdentifier::Address {
+            contract_address: contract_address.to_fixed_hex_string(),
+        },
+
+        _ => {
+            return Err(anyhow!(
+                "Exactly one of class_hash or contract_address must be provided."
```

Comment:
This is not a recoverable error, clap logic for args verification should now prevent this arm from ever being reached.

```suggestion
            unreachable!(
                "Exactly one of class_hash or contract_address must be provided."
```

Reply:
Fixed